File: 01_ctrlGPIO/04_pir/pir.py
import RPi.GPIO as GPIO

GPIO.setmode(GPIO.BCM)

GPIO.setup(23, GPIO.OUT)
GPIO.setup(24, GPIO.OUT)
GPIO.setup(18, GPIO.IN )

GPIO.output(23, False)
GPIO.output(24, False)

print "PIR Sensor Test"

try:
  while True:
    if GPIO.input(18) == 0:
      GPIO.output(23, False)
      GPIO.output(24, True )
    else:
      GPIO.output(23, False)
      GPIO.output(24, True )

except KeyboardInterrupt:
  GPIO.cleanup()
  print "\n", "Program Terminatepd!"

File: 01_ctrlGPIO/04_pir/pir.c
#include <stdio.h>
#include <wiringPi.h>

#define PIR 1
#define LED1 4
#define LED2 5

int main(void)
{
  printf("PIR Sensor Test\n");
  
  wiringPiSetup( );
  
  pinMode(PIR,  INPUT );
  pinMode(LED1, OUTPUT);
  pinMode(LED2, OUTPUT);
  
  while(1)
  {
    if( digitalRead(PIR) == 0 )
    {
      digitalWrite(4, 0);
      digitalWrite(5, 1);
    }
    else
    {
      digitalWrite(4, 1);
      digitalWrite(5, 0);
    }
  }
  return 0;
}